app/src/main/res/values/strings.xml

```diff
@@ -271,7 +271,7 @@
 
     <!-- User settings -->
 
-    <string name="pref_header_user">User information</string>
+    <string name="pref_header_user">User info</string>
```

Comment:
Why change "user information" to "user info"? I'm unsure if I like the usage of the shortcut, especially because the full wording fits nicely and I consider "info" to look weird in most cases. Any language reasons for this or just a personal preference thing?

Reply:
Fewer words, less wall of text. Results in less induced stress on part of the user.

Reply:
Yeah so no, the header should stay as it is, in my opinion.